### ISSUE TITLE
loss curve

### ISSUE THREAD

**Issue by ZhouQianang** [autonomousvision/unimatch#50]
Very good job! Could you provide a loss curve from a regular training session? I'm using the same model on another dataset, but the EPE is decreasing slowly.

**Comment by ZhouQianang** [comment]
Hello? This is the first 2000 steps of my training loss. Is this decline rate normal? I am using gmflow_scale1_train.sh with batch_size=8.

**Comment by ZhouQianang** [comment]
Hello? This is the first 2000 steps of my training loss. Is this decline rate normal? I am using gmflow_scale1_train.sh with batch_size=8, and my dataset is a sparse flow dataset.
step: 000100     epe: 32.581
step: 000200     epe: 17.243
step: 000300     epe: 12.278
step: 000400     epe: 10.447
step: 000500     epe: 9.192
step: 000600     epe: 7.654
step: 000700     epe: 6.763
step: 000800     epe: 6.227
step: 000900     epe: 5.693
step: 001000     epe: 5.022
step: 001100     epe: 4.580
step: 001200     epe: 4.256
step: 001300     epe: 3.908
step: 001400     epe: 3.654
step: 001500     epe: 3.544
step: 001600     epe: 3.247
step: 001700     epe: 3.109
step: 001800     epe: 2.928
step: 001900     epe: 2.851
step: 002000     epe: 2.692
